examples/pytorch/logging/pytorch_log_model.ipynb

```diff
@@ -0,0 +1,440 @@
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "4525800f-012c-4203-84d2-cb4017dc3d93",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "Start an mlflow run and see how our model performs!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "92645e70-05dd-47ea-a459-d4ca3726b77c",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 1, Loss: 1.2803228348493576, Accuracy: 0.0234375\n",
+      "Epoch 2, Loss: 1.2803228348493576, Accuracy: 0.0234375\n",
+      "Epoch 3, Loss: 1.2803228348493576, Accuracy: 0.0234375\n",
+      "Epoch 4, Loss: 1.2803228348493576, Accuracy: 0.0234375\n",
+      "Epoch 5, Loss: 1.2803228348493576, Accuracy: 0.0234375\n",
+      "Model training and logging complete.\n"
+     ]
+    }
+   ],
+   "source": [
+    "mlflow.set_experiment(\"iris_classification_pytorch\")\n",
+    "\n",
+    "# Start an MLflow run\n",
+    "with mlflow.start_run() as run:\n",
+    "    accuracy_metric = torchmetrics.Accuracy(\n",
+    "        task=\"multiclass\", num_classes=3\n",
+    "    )  # Instantiate the Accuracy metric\n",
+    "\n",
+    "    for epoch in range(5):  # number of epochs\n",
+    "        total_loss = 0\n",
+    "        total_accuracy = 0\n",
+    "\n",
+    "        for inputs, labels in train_loader:\n",
+    "            outputs = model(inputs)\n",
+    "            curr_loss = loss(outputs, labels)\n",
+    "            curr_loss.backward()\n",
+    "\n",
+    "            total_loss += curr_loss.item()\n",
+    "\n",
+    "            # Calculate accuracy using torchmetrics\n",
+    "            _, preds = torch.max(outputs, 1)\n",
+    "            total_accuracy += accuracy_metric(preds, labels).item()\n",
+    "\n",
+    "        avg_loss = total_loss / len(train_loader)\n",
+    "        avg_accuracy = total_accuracy / len(train_loader)\n",
+    "\n",
+    "        print(f\"Epoch {epoch + 1}, Loss: {avg_loss}, Accuracy: {avg_accuracy}\")\n",
+    "        mlflow.log_metric(\"loss\", avg_loss, step=epoch)\n",
+    "        mlflow.log_metric(\"accuracy\", avg_accuracy, step=epoch)\n",
+    "\n",
+    "    # Log the PyTorch model with the signature\n",
+    "    mlflow.pytorch.log_model(model, \"model\", signature=signature)\n",
+    "\n",
+    "    # Log parameters\n",
+    "    mlflow.log_param(\"epochs\", 10)\n",
+    "    mlflow.log_param(\"batch_size\", 16)\n",
+    "    mlflow.log_param(\"learning_rate\", 0.001)\n",
+    "\n",
+    "print(\"Model training and logging complete.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Loading the logged model back into memory"
```

Comment:
Let's call out the function we use here: `mlflow.pytorch.load_model`, also period at the end.